### PR TITLE
If there is more than one verified number for user, use default phone number

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

File: corehq/apps/sms/tests/test_verified_number.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        self.assertIsNone(self.user4.get_verified_number())
+
+    def test_user_without_verified_number(self):
+        self.assertIsNone(self.user3.get_verified_number())
+
+    def test_user_with_one_phone_number(self):
+        self.assertEqual(self.user1.get_verified_number().phone_number, '1111')
+
+    def test_user_with_two_phone_numbers(self):
+        self.assertEqual(self.user2.get_verified_number().phone_number, '3333')
```
Comment (kkrampa) [inline corehq/apps/sms/tests/test_verified_number.py]: @gcapalbo In old code this test fails (numbers are sorted and 2222 is returned). Does this change seem reasonable to you?

**Comment by gcapalbo** [comment]
@kkrampa Can you clarify what issue this will solve? In order to get the preferred number for sending to a contact, we should be using this: https://github.com/dimagi/commcare-hq/blob/657a4e9f45045126d5c100cc6e56a3820f2011f3/corehq/apps/reminders/util.py#L252

**Comment by kkrampa** [comment]
@gcapalbo I want to be sure that user.get_verified_number() returns default number for user (if default number is verified of course). E.g. In EWS user has two numbers 233246172xxx, 0203937xxx. First is default. ```get_verified_number``` always returns second one because phone numbers are sorted in this function.

**Comment by gcapalbo** [comment]
@kkrampa Where in the ews code do you invoke `get_verified_number` directly? If you're using `get_preferred_number_for_recipient` (https://github.com/dimagi/commcare-hq/blob/657a4e9f45045126d5c100cc6e56a3820f2011f3/corehq/apps/reminders/util.py#L252), it will return the verified number that is the default one if there are two verified numbers.

**Comment by kkrampa** [comment]
@gcapalbo here: https://github.com/dimagi/commcare-hq/blob/master/custom/ewsghana/reminders/reminder.py#L22

I will made a new PR.

**Pull request closed without merging**